src/statusBarTerminal.ts

```diff
@@ -7,9 +7,10 @@ export class StatusBarTerminal {
     private _terminal: Terminal;
 
     constructor(terminalIndex: number, show: boolean, name?: string, terminal?: Terminal) {
-        this._item = window.createStatusBarItem();
+        this._item = window.createStatusBarItem(1, -10);
```

Comment:
added left alignment and changed the priority to -10 in the hopes that these will always be to the far right